api/openapi/specs/core_service.yaml

```diff
@@ -0,0 +1,331 @@
+openapi: 3.0.0
+
+info:
+  title: Enclave API
+  description: API to manipulate the enclaves
+  version: 0.1.0
+
+paths:
+
+  /enclaves/{enclave_identifier}/starlark:
+    get:
+      description: Get last Starlark run
+      parameters:
+        - $ref: './api_types.yaml#/components/parameters/enclave_identifier'
+      responses:
+        default:
+          $ref: '#/components/responses/NotOk'
+        "200":
+          description: Successful request
+          content:
+            application/json:
+              schema:
+                $ref: "./api_types.yaml#/components/schemas/StarlarkDescription"
+
+  /enclaves/{enclave_identifier}/starlark/packages:
+    post:
+      description: Uploads a Starlark package. This step is required before the package can be executed with RunStarlarkPackage
+      parameters:
+        - $ref: './api_types.yaml#/components/parameters/enclave_identifier'
+      requestBody:
+        $ref: "./api_types.yaml#/components/requestBodies/fileUploadBody"
+      responses:
+        default:
+          $ref: '#/components/responses/NotOk'
+        "200":
+          description: Success
+
+  /enclaves/{enclave_identifier}/starlark/packages/{package_id}:
+    post:
+      description: Executes a Starlark script on the user's behalf
+      parameters:
+        - $ref: './api_types.yaml#/components/parameters/enclave_identifier'
+        - $ref: './api_types.yaml#/components/parameters/package_id'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "./api_types.yaml#/components/schemas/RunStarlarkPackage"
+        required: true
+      responses:
+        default:
+          $ref: '#/components/responses/NotOk'
+        "200":
+          description: Successful request
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "./api_types.yaml#/components/schemas/StarlarkRunResponseLine"
+
+  /enclaves/{enclave_identifier}/starlark/scripts:
```

Comment:
suggestion, it could be also "/enclaves/{enclave_identifier}/starlark/module" because I single file script is a module and several modules compose a package